src/Npgsql/NpgsqlCommand.cs

```diff
@@ -195,6 +195,26 @@ public override string CommandText
         }
     }
 
+    string TracingCommandText => IsWrappedByBatch ? GetBatchFullCommandText() : CommandText;
```

Comment:
Any reason we need this as a property, rather than just in line?

Reply:
Not really, just wasn't sure how big it's going to be first